### ISSUE TITLE
using unequipall button breaks plugin

### ISSUE THREAD

**Issue by runeliteissuesmaker** [SyntaxBlitz/osrs-shattered-relics-fragment-presets#17]
plugin works until unequip all is pressed, then doesnt work after, even when reloading client

**Comment by SyntaxBlitz** [comment]
hm, I can't reproduce this on the newest version

**Comment by Greggzilla** [comment]
Yep this has happened to me too.  Presets are still saved and go green when you click on them but nothing else happens.  If I uninstall the plugin to reinstall, will I lose the presets?

**Comment by thmercwade** [comment]
i just reinstalled still have my presets luckily

**Comment by SaintWacko** [comment]
I haven't even used the Unequip All button and it's broken in the same way. Reinstalling fixed it as well
